### PR TITLE
Fix travis formatting error

### PR DESCRIPTION
**Pull request by vikrambohra**
Dear Gobblin maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [ ] My PR addresses the following [Gobblin JIRA](https://issues.apache.org/jira/browse/GOBBLIN/) issues and references them in the PR title. For example, "[GOBBLIN-XXX] My Gobblin PR"
    - https://issues.apache.org/jira/browse/GOBBLIN-1163

### Description
- [ ] Here are some details about my PR, including screenshots (if applicable):
- Fix travis yaml syntax error to start builds again 

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
- This PR is a test in itself :)

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,19 +33,25 @@ script:
...
+      - secure: nTBLE5c295gwHfsMAGLhHz3Dr8eUFlEj/T6kRyVmlQ7ouvs8V97FUznlh0KwRazG0cZ5H0iXm8JxOYac3J04pvK+wOc4BpUq4WYlgBeypo0pcBp4G+5yM3+JoA0B3YbVIDKvZ6o3EBOMQ2vK/kjhe3SpOOTO4wJmfRsCx5bnbFmNKA2C8/DXDGhzfM5tfSapbc4QVYhKxSyUpq+izYGY9BZvcGxRPXLaiIxVLex3ZGpuOtrzl8gTwVnWqlQ5yYTUUdNcdJW7H1AwIlgkKKMS9+3q0KqXrJ0EtvRgbvnWdIKh1/0rOA6i4S0XgCV4mYTDqnZWO6sg3CUnAd2hFTyRI9acSWAzifkTyZD2gfsDgvsv/JQOuTOW1hZQ2g9et+vIW0wbRvIprf+wbRsNxdE7uUpGFJ9TYZJDPG6HJwqaoqyo6rcLS7T7kE6o0EZJTMw3Rq1TaSU65Ta4hvO2muEXtpPHEHh65GS0d1zcv++QQD7pedcgpIrFFbEyQJxivwXOvfklkCzTDFHI1lqK+WC1IWyC684ax+zJW8cos7I/U8kaE3w6tU9NUsOVk/yYMbqiu1YgBzMJmBjEev3RcUAyMldpdd4M2dXefdvX0PtYuZl1UOnXlpAvlhaNYb1D1+/xGgz0G0y5qR9n4nqwemjHVDzMhLPfeRjfiTnq6FP8MkI=
+  jobs:
+    - RUN_TEST_GROUP=build
```
Comment (autumnust) [inline .travis.yml:51]: Why this is moved from env block ?

File: travis/bintrayDeploy.sh [hunk 2]
```diff
@@ -32,6 +32,7 @@ echo "Build Version: $BUILD_VERSION"
 
 echo "Pull request: [$TRAVIS_PULL_REQUEST], Travis branch: [$TRAVIS_BRANCH]"
 # release only from master when no pull request build
+
```
Comment (autumnust) [inline travis/bintrayDeploy.sh:35]: Can you remove this blank line?

File: .travis.yml (hunk 1 above)
Comment (vikrambohra) [reply .travis.yml:51]: It is still in ENV block. the new syntax puts in under jobs bucket of the env block to differentiate global variables from job level variables.

File: travis/bintrayDeploy.sh (hunk 2 above)
Comment (vikrambohra) [reply travis/bintrayDeploy.sh]: done

**Pull request closed without merging**